test/postgresDbTest.js

```diff
@@ -0,0 +1,85 @@
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const server = require('../server/index.js');
+const postgresDb = require('../postgresDb/index.js');
+
+const should = chai.should();
+
+chai.use(chaiHttp);
+
+const id = Math.floor(Math.random() * 300000);
+
+describe('Movies', () => {
+  // it('testing', (done) => {
```

Comment:
Do you still need this?

Reply:
I wrote this to test the server, but ended up querying the databases directly.  I'm not using this at the moment, but I want to keep it as a model to reference when I do write server tests.